src/log.js

```diff
@@ -955,18 +955,10 @@ export function devAssert(
  * @param {*=} opt_7 Optional argument
  * @param {*=} opt_8 Optional argument
  * @param {*=} opt_9 Optional argument
- * @return {R} The value of shouldBeTrueish.
+ * @return {T} The value of shouldBeTrueish.
+ * @throws {!Error} When `shouldBeTrueish` is falsey.
  * @template T
- * @template R :=
```

Comment:
What was `R` for?

Reply:
I think it comes from [here](https://github.com/ampproject/amphtml/pull/22788/files/38793d3b612acdb364f4c98a623cb2adc649e167#diff-4f24bb7c46eda8f63690cb849ddb0f2d).